packages/service-worker/worker/src/driver.ts

```diff
@@ -354,7 +354,7 @@ export class Driver implements Debuggable, UpdateSource {
 
     const notificationAction = action === '' || action === undefined ? 'default' : action;
 
-    const onActionClick = notification?.data?.onActionClick[notificationAction];
+    const onActionClick = (notification?.data?.onActionClick || [])[notificationAction];
```

Comment:
This avoids creating an array:
```suggestion
    const onActionClick = notification?.data?.onActionClick?.[notificationAction];
```

Reply:
Thanks!